websites/W/watchmovieshd/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+		clientId: "1013927311499874304",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameData: {
+	currTime: number;
+	duration: number;
+	paused: boolean;
+} = null;
+
+presence.on(
+	"iFrameData",
+	(data: { currTime: number; duration: number; paused: boolean }) => {
+		iFrameData = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "logo",
```

Comment:
I've been told that Imgur links are preferred for these images for easier maintenance by other developers

Reply:
Why was this resolved?